src/Cooked/MockChain/UtxoState.hs

```diff
@@ -2,47 +2,45 @@ module Cooked.MockChain.UtxoState
   ( UtxoState (..),
     UtxoPayloadSet (..),
     UtxoPayload (..),
-    holdingInState,
+    holdsInState,
   )
 where
 
 import Cooked.Skeleton (TxSkelOutDatum)
-import Cooked.Wallet (Wallet, walletAddress)
 import Data.Function (on)
 import qualified Data.List as List
 import Data.Map.Strict (Map)
 import qualified Data.Map.Strict as Map
 import qualified Ledger as Pl hiding (Value)
-import qualified Plutus.V1.Ledger.Value as Pl1
+import qualified Plutus.Script.Utils.Value as Pl
+import qualified Plutus.V2.Ledger.Api as PV2
```

Comment:
Our infamous non existent naming convention for qualified modules... In this module, we intended to name everything coming from explicit V1 `Pl1` and V2 `Pl2`. I know we have `PV2` in other places but we really should sort things out at some point. Is `PV2` the de facto convention?

Reply:
`PV2` seems to win in terms of occurrences, `Pl2` is used in the recent `UtxoSearch` module. I don't think we will ever converge :smile: